index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/162dfb3961.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/general.css" />
+    <link rel="stylesheet" href="CSS/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- Navigation -->
+    <header class="header">
+      <nav>
+        <div class="navbar">
+          <i class="fa-solid fa-bars menu"></i>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Headline -->
+    <section class="headline">
+      <div class="container headline-container">
+        <div class="headline-header">
+          <h1 class="heading-title headline-title-1">
+            Hi there. I'm Umuhan Mohamud.
+            <span class="heading-title headline-title-2"
+              >I'm a software developer</span
+            >
+          </h1>
+          <p class="heading-subtitle">
+            I can help you build a product , feature or website Look through
```

Comment:
- [ ] Please check the original design : font-family, margins, paddings, colors, and icons-size , 


- Your design 👇 

![image](https://user-images.githubusercontent.com/75126481/234868323-ad3f368e-8dc3-4f36-931a-639467a9d5e4.png)

- Original design 👇 

![image](https://user-images.githubusercontent.com/75126481/234868561-ca5fba80-3fac-4742-bfa2-5e01ddb14027.png)